### DISCUSSION TITLE
Change Login Email Link

### DISCUSSION THREAD

**Discussion by doloboyz** [jeremyevans/rodauth#335] [General]
Hello, 
I was wondering if there's a method available to change the email link, subject, and email body used in the change_login email. Don't see a method in [change_login.rdoc](https://rodauth.jeremyevans.net/rdoc/files/doc/change_login_rdoc.html). Appreciate any help!

**Comment by janko** [comment]
This is because the change_login feature doesn't send an email confirmation, that's done by the [verify_login_change](http://rodauth.jeremyevans.net/rdoc/files/doc/verify_login_change_rdoc.html) feature.

**Reply by doloboyz** [reply]
Appreciate the help!
